server/grpc/gogoreflection/fix_registration.go

```diff
@@ -3,10 +3,9 @@ package gogoreflection
 import (
 	"reflect"
 
+	_ "github.com/cosmos/cosmos-proto"        // look above
```

Comment:
look above comment doesn't not work if there is no comment above :D
can we duplicate it?